### PR TITLE
Add Ordering Rules

### PR DESCRIPTION
**Pull request by ikeinyyo**

### CODE REVIEW

**Review by lbarba — APPROVED** [review]

File: Documents/OrderingRules.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+According to the [StyleCop Rules Documentation](http://stylecop.soyuz5.com/Ordering%20Rules.html) the ordering is as follows.
+
+Within a class, struct or interface: (SA1201 and SA1203)
+
+* Constant Fields
+* Fields
+* Constructors
+* Finalizers (Destructors)
+* Delegates
+* Events
```
Comment (kenny-reyes) [inline Documents/OrderingRules.md]: Events declaration is a field, but there should be with fields

**Review by kenny-reyes — DISMISSED** [review]
some considerations that I don't agree with your order

**Review by danimart1991 — APPROVED** [review]

**Review by pgonzalezplain — APPROVED** [review]

**Review by psanchezelena — APPROVED** [review]

**Review by franmolmedo — APPROVED** [review]
I prefer another order relative to delegates mainly. If a found an appropiate example i would submit another pr

File: Documents/OrderingRules.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+## Rules
+
+Within a class, struct or interface
+
+* Constant Fields
+* Fields
+* Constructors
+* Finalizers (Destructors)
+* Delegates
+* Enums
```
Comment (lbarba) [inline Documents/OrderingRules.md:16]: I prefer to have every Enum type in its own file. I think we should not include more that one Class or Enum in a file.

Comment (ikeinyyo) [reply Documents/OrderingRules.md:16]: Yes. This is only for a sorting in the same file. Nested classes or enums for me are ugly.

**Comment by ikeinyyo** [comment]
@franmolmedo It's perfect! That is the idea :)

**Review by acnagrellos — APPROVED** [review]

File: Documents/OrderingRules.md (hunk 1 above)
Comment (danimart1991) [reply Documents/OrderingRules.md]: **Third reviewer.** Events are declared LIKE fields, but they AREN'T fields. The thing here is try to follow Microsoft rules. And in Microsoft code, Events goes between Properties and Methods.

In other order, ALL the team accept this except you. So I'm not only the Third Reviewer, I'm the voice of the team: **_"This PR must be merged, accepted and used."_**

**Pull request merged**